app/views/gsa18f/_proposal_mail.html.haml

```diff
@@ -2,7 +2,8 @@
   %table.w-container.main-container{ width: "800" }
     %tr
       %td
-        %h1.c2_header= "#{title} : #{proposal.name}"
+        %h1.c2_header
+          = link_to proposal.name, proposal_url(proposal),  { target: "C2" }
```

Comment:
I just removed the `target` from the URL in https://github.com/18F/C2/pull/949 because it didn't seem to be doing anything - do you know why we have it in this link?